.ci/docker/node-playwright/Dockerfile

```diff
@@ -9,9 +9,26 @@ FROM ubuntu:bionic
 ARG NODEJS_VERSION
 
 # 1. Install node
-RUN apt-get -qq update && apt-get -qq install -y curl && \
-    curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
-    apt-get -qq install -y nodejs
+RUN \
+  apt-get update > /dev/null \
+  && apt-get install -y --no-install-recommends \
+    "ca-certificates=*" \
+    "curl=*" \
+    "gnupg=*" \
+    "lsb-release=*" \
```

Comment:
Aren't we mixing here a few concerns? The one regarding bumping the GitHub actions and using something else for configuring the dockerfile?